presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -1058,7 +1059,7 @@ private void addArrayTimestampTestIfSupported(DataTypeTest tests, DataType<List<
         };
     }
 
-    @Test(dataProvider = "testTimestampWithTimeZoneDataProvider")
```

Comment:
> Rename misleading testTimestampWithTimeZoneDataProvider

It was following `<test name>DataProvider` convention.
What do you find misleading about that?


(im still ok with the removal)